infrateam_first_responder.md

```diff
@@ -60,7 +60,7 @@ To complete the cycle, and ensure QA has the common environment, deploy there as
 
 ### Notify Coverage for Following Week
 
-Set Slack reminders in `#dlss-infrastructure` for next week's Monday morning. The reminders should indicate who is first responder and who is on deck for that week, and should be set for 3 am Pacific time/6 am Eastern, so that the east coast early risers don't have to wait for it.
+Set Slack reminders in `#dlss-infrastructure` for next week's Monday morning. The reminders should indicate [who is first responder](https://docs.google.com/spreadsheets/d/13TJR93Yc9_eF5B7w4XDx6ggG_wb3aLkgCHjpLwmHPBA/edit#gid=1409336640) and who is on deck for that week, and should be set for 3 am Pacific time/6 am Eastern, so that the east coast early risers don't have to wait for it.
```

Comment:
suggest leaving off the `#gid` param in the URL so it just goes to the default (first) worksheet instead of the 2023.